### ISSUE TITLE
'< '> overwritten

### ISSUE THREAD

**Issue by BertrandSim** [kana/vim-textobj-user#70]
Hi Kana,

I noticed that vim-textobj-user overwrites the last visual selection, for example, with `yil` and vim-textobj-line. 

It seems that this issue has been fixed before #55 , but in the latest version (0.7.6) this issue seems to be back again

**Comment by kana** [comment]
See https://github.com/kana/vim-textobj-user/issues/59 for the details.

**Comment by BertrandSim** [comment]
Maybe I am missing something, but I don't see how vim-surround is related to this issue.

**Comment by BertrandSim** [comment]
Indeed, I had missed something.  While #55 fixed this issue, it resulted in this plugin being incompatible with custom operators that use getchar(), such as vim-surround. As I understand it, the PR #55 was reverted to allow compatibility with custom operators, at the cost of overwriting visual marks '< and '> .
